src/components/Map.tsx

```diff
@@ -102,7 +103,12 @@ const Map = ({
   }, [onMapMoved])
 
   const centerUserLocation = useCallback(() => {
-    const hasCoords = userCoords && userCoords.longitude && userCoords.latitude
+    const hasCoords =
```

Comment:
Null and undefined aren't numbers so I think we can shorten this to 
```ts
 const hasCoords =
      userCoords &&
      isNumber(userCoords.longitude) &&
      isNumber(userCoords.latitude
```

Reply:
updated